### PR TITLE
Small fixes

### PR DESCRIPTION
**Pull request by adisuciu**

### CODE REVIEW

**Comment by DanielGuramulta** [comment]
The siggen nr of periods is already on master f879e010da9619dbbb3a7a697f32a8123b40af28. Remove the duplicate commit

File: src/network_analyzer.cpp [hunk 1]
```diff
@@ -1178,6 +1179,7 @@ void NetworkAnalyzer::goertzel()
 		}
 	}
 
+	Q_EMIT sweepStart();
```
Comment (DanielGuramulta) [inline src/network_analyzer.cpp:1182]: Is this sweepStart signal used anywhere?

File: src/network_analyzer.cpp [hunk 2]
```diff
@@ -702,6 +702,7 @@ NetworkAnalyzer::NetworkAnalyzer(struct iio_context *ctx, Filter *filt,
 
 	});
 
+	connect(this,SIGNAL(sweepDone()), ui->xygraph ,SLOT(reset()));
```
Comment (DanielGuramulta) [inline src/network_analyzer.cpp:705]: 3 different styles for separating with `,` used in one line. `a,b`, `a, b` and `a ,b`. Just use one. (prefered `a, b`)

**Comment by DanielGuramulta** [comment]
There's also a merge commit in here, remove it and rebase this branch with master

File: src/network_analyzer.cpp (hunk 1 above)
Comment (adisuciu) [reply src/network_analyzer.cpp:1182]: It is now :stuck_out_tongue_winking_eye:

File: src/network_analyzer.cpp (hunk 2 above)
Comment (adisuciu) [reply src/network_analyzer.cpp]: This was sloppy. Fixed.

**Comment by adisuciu** [comment]
Rebased

**Review by DanielGuramulta — APPROVED** [review]
Looks good!

**Pull request merged**